docs/sphinx/raja_license.rst

```diff
@@ -13,7 +13,7 @@
 RAJA License
 ===================================
 
-RAJA version 0.3.1
+RAJA version 0.4.0
```

Comment:
We should update the copyright date here to match the stuff in other files; i.e., 2016-2017.  Having a sed script to easily propagate these change throughout all files would be good to add.

Reply:
Good idea!